src/duqtools/submit.py

```diff
@@ -165,11 +174,11 @@ def submit(*, force: bool, max_jobs: int, schedule: bool, array: bool,
         that needs to be resubmitted, or the shortname
     """
     if not cfg.submit:
-        raise Exception('submit field required in config file')
+        raise SubmitError('Submit field required in config file')
```

Comment:
This will never happen, as `cfg.submit` is not optional (it will always exist), either we remove this check, or make `cfg.submit` optional, up to you

Reply:
Yeah, I figured this was the case. Since it was already there left it in. I will remove it now 👍